examples/access-control-migration/README.md

```diff
@@ -1,6 +1,306 @@
 # @loopback/example-access-control-migration
 
-This is the basic tutorial for getting started with Loopback 4!
+This example is migrated from
+[loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control),
+and uses the authentication and authorization system in LoopBack 4 to implement
+the access control.
+
+# App Scenario
+
+This example is a donation system. Users can view or make donations to projects
+based on their roles. A project is owned by a user, and a user can create a team
+to involve other users as team members. The system has an administration.
+
+Here is the diagram that describes the models:
+
+![models](./imgs/scenario-model.png)
+
+And the Project endpoints with their allowed roles:
```

Comment:
Maybe we can make it more clear here:
`In this example, we have three role: ...... And the following operations can be processed by different roles.`